### PR TITLE
Wire in hostname configuration option

### PR DESCRIPTION
**Pull request by cybertron**
A variable to allow configuring the hostnames of the Ironic nodes was added in #234. This wires it into the sample config file so it is easier to consume and document.

### CODE REVIEW

**Comment by cybertron** [comment]
/ok-to-test

Let's see if this works now that I made my metal3-io membership public...

File: host_cleanup.sh [hunk 1]
```diff
@@ -58,3 +58,7 @@ if [[ $OS == "centos" || $OS == "rhel" ]]; then
       sudo rm -f /etc/sysconfig/network-scripts/ifcfg-baremetal || true
   fi
 fi
+
+# There is a dependency issue between RDO and EPEL that means we need to clean
+# these repos up or subsequent runs will fail.
+sudo rm -f /etc/yum.repos.d/delorean*
```
Comment (maelk) [inline host_cleanup.sh]: This is unrelated to the PR title and desc. Please remove or update the PR

**Comment by maelk** [comment]
This needs a rebase to pass the tests @cybertron

**Comment by cybertron** [comment]
Rebase done. Thanks for the heads up.

**Comment by cybertron** [comment]
> /lgtm . Please update the try-it.md on the website too.

Already proposed: https://github.com/metal3-io/metal3-io.github.io/pull/152
